Makefile

```diff
@@ -9,3 +14,19 @@ check:
 
 test: 
 	Rscript  -e "library(testthat); test()"
+
+docker_dev_image:
+	@docker run -it --name irkernel-build \
+	$(BASE_IMAGE) bash -c 'Rscript -e "install.packages(c('\''devtools'\''), repos = c('\''http://irkernel.github.io/'\'', '\''http://cran.rstudio.com'\''))" && \
+	                        Rscript -e "library(devtools)" \
+	                        -e "install_github('\''IRkernel/repr'\'')" \
+	                        -e "install_github('\''IRkernel/IRdisplay'\'')"'
+	@docker commit irkernel-build $(DEV_IMAGE)
+	@docker rm -f irkernel-build
```

Comment:
https://github.com/jupyter-incubator/dashboards/blob/master/Makefile#L21

and

https://github.com/jupyter-incubator/dashboards/blob/master/Dockerfile.dev

for reference.